include/arch/arm/arch/object/vcpu.h

```diff
@@ -78,11 +78,11 @@ struct vcpu {
     word_t regs[seL4_VCPUReg_Num];
     bool_t vppi_masked[n_VPPIEventIRQ];
 #ifdef CONFIG_VTIMER_UPDATE_VOFFSET
-    /* vTimer is 8-bytes wide and has same alignment requirement.
-     * The struct will remain packed on 32-bit platforms when n_VPPIEventIRQ
-     * is odd, but were it to become even, an extra word of padding will be
```

Comment:
I'd suggest changing it to "but *since it is currently* even, and extra word of padding *is* necessary", so if another word is added we can go back easily. I have difficulty reading the new proposed version.

Reply:
Happy to tweak, but it's not currently even. The previous comment was incomplete in the sense that the padding isn't determined by `n_VPPIEventIRQ` alone -- it's dependent on whether the sum of the length of the two arrays is even or odd. It's currently odd, which means we need padding. If the sum is even we don't.

So maybe the following?
```
    /* vTimer is 8-bytes wide and has same alignment requirement.
     * If the sum of  n_VPPIEventIRQ and seL4_VCPUReg_Num is even,
     * we do not need extra padding. If the sum is odd we do. It currently
     * is odd, so the extra padding above is necessary for the struct to
     * remain packed on 32 bit platforms.
```

Reply:
(comment now updated as above)